app/assets/javascripts/tylium/modules/issues/tag-input.js.coffee

```diff
@@ -5,3 +5,8 @@ document.addEventListener "turbolinks:load", ->
     $span = $("#issues_editor .dropdown-toggle span.tag")
     $span.html($target.html())
     $span.css("color", $target.css("color"))
+
+    # Trigger local auto save and store selected tag value in local cache
```

Comment:
Do we really need this check? What is the benefit of using this over adding `$span.trigger('textchange')` in L8?

Reply:
Fixed, I forgot to remove the check as previously it was just for when creating new resource